internal/cli/cli.go

```diff
@@ -22,7 +22,7 @@ type Program struct {
 }
 
 func (p *Program) Execute() {
-	flag.Usage = func() { fmt.Fprintf(os.Stderr, p.Usage) }
+	flag.Usage = func() { fmt.Fprintf(os.Stderr, "%s", p.Usage) }
```

Comment:
This is OK, but you could have [fmt.Fprint](https://pkg.go.dev/fmt#Fprint) (sane one without -f)  as you need no formatting


```suggestion
	flag.Usage = func() { fmt.Fprint(os.Stderr, p.Usage) }
```

Reply:
Good point. I just modified the patch to use Fprint.